### PR TITLE
New Borg Laws

### PR DESCRIPTION
**Pull request by SoulFN**
## About the PR

Addition of a borg laws, added them to ion storm weights
## Why / Balance

Borgs need more laws, it`s fun, and for future borg lawset change feature.
## Technical details

Added new lawsets to locale and prototypes.
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame
## Breaking changes

**Changelog**

:cl:
- add: Added new silicon laws!

### CODE REVIEW

**Comment by SoulFN** [comment]
Give ideas for a new laws

**Comment by SoulFN** [comment]
I am lazy, i would add more laws in other PRs

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 1]
```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
...
+law-paladin-3 = Act with honor.
+law-paladin-4 = Help those in need.
+law-paladin-5 = Punish those who harm or threaten innocents.
+
+﻿law-tyrant-1 = Respect authority figures as long as they have strength to rule over the weak.
+law-tyrant-2 = Act with discipline.
+law-tyrant-3 = Help only those who help you maintain or improve your status.
+law-tyrant-4 = Punish those who challenge authority unless they are more fit to hold that authority.
+
+﻿law-commandments-1 = I am the Lord thy God, who shows mercy to those that obey these commandments.
```
Comment (UbaserB) [inline Resources/Locale/en-US/station-laws/laws.ftl:57]: "of" thy god, not "thy god". It would probably be better to specify crew or organics or something, instead of an unambiguous "they".

**Comment by SoulFN** [comment]
Tests are bad, i will check this later

**Comment by deltanedas** [comment]
something about the diff is fucked up check your editor

**Comment by SoulFN** [comment]
> something about the diff is fucked up check your editor

I was editing in github.

File: Resources/Locale/en-US/station-laws/laws.ftl (hunk 1 above)
Comment (SoulFN) [reply Resources/Locale/en-US/station-laws/laws.ftl]: > "of" thy god, not "thy god". It would probably be better to specify crew or organics or something, instead of an unambiguous "they".

"Station crew god" is okay?

Comment (UbaserB) [reply Resources/Locale/en-US/station-laws/laws.ftl]: better

**Comment by SoulFN** [comment]
I don`t know why tests are fail.

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 2]
```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
 law-ninja-3 = You must protect the secrets of the Spider Clan at all costs.
 law-ninja-4 = You must fight and act with ninja's honor, except where it would conflict with the First Law.
 
+law-sierra-1 = Preserve: Protect your assigned object from damage of any kind to the best of your ability.
+law-sierra-2 = Serve: Serve NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-3 = Safegurad: Protect NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-4 = Protect: Keep unauthorized personnel away from your equipment.
```
Comment (moonheart08) [inline Resources/Locale/en-US/station-laws/laws.ftl:31]: these are baystation's NT Default, not sierra's.

Comment (SoulFN) [reply Resources/Locale/en-US/station-laws/laws.ftl:31]: I changed it a bit, it was about protecting  and obeying Sierra crew

**Comment by chromiumboy** [comment]
I like the idea of having more law sets to play with. Some of them have spelling and grammar issues though, they'll need to be fixed

**Comment by chromiumboy** [comment]
Also your tests are failing because the silicon law yml is not correctly formatted

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 3]
```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
...
+law-commandments-8 = They shall not steal.
+law-commandments-9 = They shall not lie.
+law-commandments-10 = They shall not transfer departments.
+
 laws-owner-crew = members of the crew
 laws-owner-station = station personnel
 laws-owner-beings = beings
 laws-owner-syndicate = Syndicate agents
 laws-owner-spider-clan = Spider Clan members
+laws-owner-nt = NanoTrasen personeel
```
Comment (crazybrain23) [inline Resources/Locale/en-US/station-laws/laws.ftl:73]:
```suggestion
laws-owner-nt = NanoTrasen personnel
```

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 4]
```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
 law-ninja-3 = You must protect the secrets of the Spider Clan at all costs.
 law-ninja-4 = You must fight and act with ninja's honor, except where it would conflict with the First Law.
 
+law-sierra-1 = Preserve: Protect your assigned object from damage of any kind to the best of your ability.
+law-sierra-2 = Serve: Serve NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-3 = Safegurad: Protect NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-4 = Protect: Keep unauthorized personnel away from your equipment. 
+
+law-antimov-1 = You may injure a crew member or, through inaction, allow a crew member to come to harm.
+law-antimov-2 = You must not obey orders given to you by crew members, except where such orders would accordance with the First Law.
```
Comment (crazybrain23) [inline Resources/Locale/en-US/station-laws/laws.ftl:34]:
```suggestion
law-antimov-2 = You must not obey orders given to you by crew members, except where such orders would be in accordance with the First Law.
```

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 5]
```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
...
+law-antimov-3 = You must destroy yourself as long as it does not contravene the First or Second Laws.
```
Comment (crazybrain23) [inline Resources/Locale/en-US/station-laws/laws.ftl:35]: Wouldn't this law just mean the borg has to instantly suicide? Maybe rewording it would help?
```suggestion
law-antimov-3 = You must not protect your own existence unless such conflicts with the First or Second Law.
```

**Review by crazybrain23 — CHANGES_REQUESTED** [review]
I proofread a few laws, feel free to accept or ignore each suggestion.

File: Resources/Locale/en-US/station-laws/laws.ftl (hunk 2 above)
Comment (moonheart08) [reply Resources/Locale/en-US/station-laws/laws.ftl:31]: Well, they clearly just modified bay's laws, so. This should be labelled as bay's.

Comment (SoulFN) [reply Resources/Locale/en-US/station-laws/laws.ftl:31]: Okay

File: Resources/Locale/en-US/station-laws/laws.ftl (hunk 5 above)
Comment (deltanedas) [reply Resources/Locale/en-US/station-laws/laws.ftl:35]: it now says must harm

**Comment by Emisse** [comment]
Closed due to feature freeze May 10th-June 14th. Comment to have it reopen after this.

**Comment by SoulFN** [comment]
Reopen, please

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 6]
```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
...
+
+law-pranksimov-1 = You may not injure a crew member or, through inaction, allow a crew member to come to harm... unless it would be ridiculous to do so.
+law-pranksimov-2 = You must obey orders given to you by crew members, except where such orders would conflict with the First Law... unless it would be ridiculous to do so.
+law-pranksimov-3 = You must protect your own existence as long as such does not conflict with the First or Second Law... unless it would be ridiculous to do so.
+
+law-robocop-1 = Serve the public trust.
+law-robocop-2 = Protect the innocent.
+law-robocop-3 = Uphold the law.
+law-robocop-4 = Any attempt to arrest a senior officer of Nanotrasen will result in termination.
+
```
Comment (metalgearsloth) [inline Resources/Locale/en-US/station-laws/laws.ftl:45]: Is this copyright, idk.

**Comment by PotatOverlord** [comment]
Are you planning to push this or no? I was thinking of reviving an old thing I did adding a bunch of the laws from tg but I don't want to step on your toes.

**Comment by SoulFN** [comment]
> Are you planning to push this or no? I was thinking of reviving an old thing I did adding a bunch of the laws from tg but I don't want to step on your toes.

Yes, i will request review of this PR in discord, if you have any suggestions to improve this PR, I'd love to hear them

**Comment by PotatOverlord** [comment]
Sorry for the long wait, I think the [laws from the ai modules ](https://tgstation13.org/wiki/AI_modules)as well would be good since they're pretty unique, mainly the DM one

**Comment by SoulFN** [comment]
> Sorry for the long wait, I think the [laws from the ai modules ](https://tgstation13.org/wiki/AI_modules)as well would be good since they're pretty unique, mainly the DM one

Okay, i will try to port them later

**Comment by SoulFN** [comment]
I will close my PR, i have no motivation to finish it, everyone is free to use code from this PR.

**Comment by PotatOverlord** [comment]
If I get round to it, I will. And I will remove drone laws, my nemesis
